core-bundle/contao/themes/flexible/styles/main.css

```diff
@@ -742,7 +742,7 @@ h2.sub_headline {
 	border-bottom: 1px solid var(--content-border);
 	position: sticky;
 	top: 0;
-	z-index: 3;
+	z-index: 3; /* Above TinyMCE */
```

Comment:
There is also an _Above TinyMCE_ comment in line 853, however, it only uses `z-index: 2` and not `3`.  So either the value is wrong here or the comment is incomplete.

Reply:
I was simply referring to https://github.com/contao/contao/pull/6747#discussion_r1460344690 and commented every `z-index` in the file.

With the outer `.jump-targets` having a higher `z-index` already, I am unsure if the ancestor needs to have 2 or if 1 would be sufficient.

Reply:
Yes, but why do we use `z-index: 3` here and only `z-index: 2` in line 853? According to the comment, both do the same thing.